src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -245,9 +245,9 @@ private bool TryPopulateOpenTextBufferManagerForBuffer(ITextBuffer buffer, IEnum
                 }
             }
 
-            var documentSupportsFeatureService = _workspace.Services.GetService<IDocumentSupportsFeatureService>();
+            var documentSupportsFeatureService = _workspace.Services.GetService<ITextBufferSupportsFeatureService>();
 
-            if (!_openTextBuffers.ContainsKey(buffer) && documents.All(d => documentSupportsFeatureService.SupportsRename(d)))
+            if (!_openTextBuffers.ContainsKey(buffer) && documentSupportsFeatureService.SupportsRename(buffer))
             {
                 _openTextBuffers[buffer] = new OpenTextBufferManager(this, buffer, _workspace, documents, _textBufferFactoryService);
```

Comment:
>documents [](start = 95, length = 9)

We can remove `documents` parameter from `OpenTextBufferManager` constructor as well.